src/App.js

```diff
@@ -1,31 +1,83 @@
-import React, {Component} from 'react'
+import React from 'react'
 import {HashRouter as Router, Route} from 'react-router-dom'
-import {App as GrommetApp, Article, Footer} from 'grommet'
-import {I18nextProvider} from 'react-i18next'
-import {Header} from './components'
+import {Grommet, Box, Footer} from 'grommet'
 import {FAQ, Home, PoAForm} from './screens'
 
 import {FAQ_PATH, FORM_PATH, HOME_PATH} from './paths'
-import i18n from './helpers/i18n'
-
-class App extends Component {
-  render() {
-    return (
-      <Router>
-        <GrommetApp>
-          <I18nextProvider i18n={i18n}>
-            <Article pad={{horizontal: 'large'}}>
-              <Header />
-              <Route exact path={FAQ_PATH} component={FAQ} />
-              <Route exact path={FORM_PATH} component={PoAForm} />
-              <Route exact path={HOME_PATH} component={Home} />
-              <Footer />
-            </Article>
-          </I18nextProvider>
-        </GrommetApp>
-      </Router>
-    )
+import POAHeader from './components/POAHeader'
+
+const POA_CORAL_BLUE = "#679ba1"
```

Comment:
We probably want to organize this as part of theme/ styles files and export the values so they can be used in other parts of the app.

Reply:
Makes sense, I'll move the theme into its own file.